book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rentals << rental
```

Comment:
`add_rental` should take `person` and `date` as a parameter. A new rental should be initialized using the keyword `new` and the book itself should be sent as a parameter to create the new rental achieving the **has-many** association. Like this: `Rental.new(date, self, person)`. Kindly update your code accordingly.